concent_api/core/models.py

```diff
@@ -11,7 +11,7 @@ class Message(Model):
     type        = PositiveSmallIntegerField()
     timestamp   = DateTimeField()
     data        = BinaryField()
-    task_id     = IntegerField(null = True)
+    task_id     = CharField(max_length = 255, null = True)
```

Comment:
How this can break support for non-numeric IDs. Have you meant numeric IDs ? Wasn't that the purpose of this change ?

Reply:
>  Have you meant numeric IDs ?

Yes, sorry for the confusion.

> Wasn't that the purpose of this change ?

The purpose was to add support for non-numeric IDs. Your change does this but at the same time breaks numeric IDs in a way.

Looks like Golem lets you use anything as ID. And I did not see any string conversions there so I suspect that to Golem ids like `666` and `'666'` are different things since that's how Python works.

After your change we "support" both of these in the sense that there's no exception. But Django converts anything you assign to a `CharField` to string and `filter()` will find `'666'` if you give it `666`. So we don't really support numeric IDs any longer. We just convert them to strings (which Golem does not do).

Reply:
Right, but I thought that is how we want it to work. So lets wait for Darek's response to use only strings which I think is best solution in that case, otherwise we would have to store extra info on the original type or use separate field for ints and strings. 
Btw. I don't see any casting of task_id in golem_messages, is there any option, apart from asking Darek, which you already did, to check how it works on Golem side ?